### DISCUSSION TITLE
LoRa baud rates question

### DISCUSSION THREAD

**Discussion by mattbk** [SQ9MDD/TTGO-T-Beam-LoRa-APRS#58] [General]
To what does the "Speed [baud]" field refer in the www interface? I assume this is the speed of LoRa transmission. Is one better than the other for various applications? From other things I am reading about LoRa, other rates are available, so why the limit?

I also found out through trial and error that different baud rates on different devices can't communicate with each other. So this could be used as a way to channelize traffic?

**Comment by M0IGA** [comment]
Quick and IMO the best explanation is here: [LoRa bitrate calculator and understanding LoRa parameters](https://unsigned.io/understanding-lora-parameters/)
**Generally higher speed lower range.**
The 300bps speed was chosen by Austrian HAMs and propagated in Europe for the longest distances. They use iGates only, so the long time of a frame transmission isn't a big problem. 
Our view of lora use was different. We want to build the fully working aprs network with digis, aliases, etc. so we chose a similar speed at the expense of range. There is no any technical reason to use a different speed if it's needed.
